graylog2-web-interface/src/components/search/MessageFields.jsx

```diff
@@ -19,7 +19,7 @@ const MessageFields = React.createClass({
 
   _formatFields(fields, showDecoration) {
     if (!showDecoration || !this.props.message.decoration_stats) {
-      return Object.keys(fields).sort().map(key => <MessageField {...this.props} fieldName={key} value={fields[key]} />);
+      return Object.keys(fields).sort().map(key => <MessageField key={key} {...this.props} fieldName={key} value={String(fields[key])} />);
```

Comment:
The only problem I see with relaxing the prop, is that some types may not render properly, like booleans. As at the moment we don't do anything special with the value, I don't see any problem with converting it to a string to be honest.